### PR TITLE
detection tutorials [WIP]

### PR DESCRIPTION
**Pull request by wtomin**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Tutorials on how to train text detection models on custom datasets.

### CODE REVIEW

**Review by zhtmike — APPROVED** [review]

File: docs/en/tutorials/training_detection_custom_dataset.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+optimizer:
+  opt: SGD
+  filter_bias_and_bn: false
+  momentum: 0.9
+  weight_decay: 1.0e-4
+```
+It uses `SGD` optimizer (in `mindocr/optim/optim.factory.py`) and `polynomial_decay` (in `mindocr/scheduler/scheduler_factory.py`) as the learning scheduler. The loss function is `L1BalancedCELoss` (in `mindocr/losses/det_loss.py`) and the evaluation metric is `DetMetric` ( in `mindocr/metrics/det_metrics.py`).
+
+
+## 3. Model Training, Evaluation, and Inference
```
Comment (SamitHuang) [inline docs/en/tutorials/training_detection_custom_dataset.md:223]: Please do not skip to dbnet training in 3. This is the critical part in this tutorial. Rec tutorial will also rewrite this part.

Comment (wtomin) [reply docs/en/tutorials/training_detection_custom_dataset.md:223]: OK，sure. I will revise it accordingly.

**Review by hadipash — APPROVED** [review]

File: docs/en/tutorials/training_detection_custom_dataset.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+Currently, MindOCR detection network supports two input formats, namely
+- `Common Dataset`：A file format that stores images, text bounding boxes, and transcriptions. An example of the target file format is:
+``` text
+img_1.jpg\t[{"transcription": "MASA", "points": [[310, 104], [416, 141], [418, 216], [312, 179]]}, {...}]
+```
+
+It is read by [DetDataset](../../../mindocr/data/det_dataset.py). If your dataset is not in the same format as the example format, see [instructions](../../../tools/dataset_converters/README.md) on how convert different datasets' annotations into the supported format.
+
+- `SynthTextDataset`: A file format provided by [SynthText800k](https://github.com/ankush-me/SynthText), consisting of `imnames`(image names), `wordBB`(word-level bounding-boxes), `charBB`(character-level bounding boxes), and `txt` (text strings). It is read by [SynthTextDataset](../../../mindocr/data/det_dataset.py). Users can take `SynthTextDataset` as a reference to write their custom dataset class.
```
[the remark this answers is not in the document]
Comment (wtomin) [reply docs/en/tutorials/training_detection_custom_dataset.md]: I added this link to this tutorial

**Review by SamitHuang — APPROVED** [review]
